experian-fraud-stub/src/main/resources/GenericResponse/fraud-ex--peps.json

```diff
@@ -0,0 +1,295 @@
+{
+  "responseHeader": {
+    "requestType": "PepSanctions01",
+    "clientReferenceId": "7f1e1b09-e72f-439e-aea8-c77c1bc5ca14",
+    "expRequestId": "RB000716196739",
+    "messageTime": "2022-06-24T14:13:33Z",
+    "overallResponse": {
+      "decision": "REFER",
+      "decisionText": "Continue and Investigate",
+      "decisionReasons": [
+        "Not Authenticated"
+      ],
+      "recommendedNextActions": [],
+      "spareObjects": []
+    },
+    "responseCode": "R0201",
+    "responseType": "INFO",
+    "responseMessage": "Workflow Complete.",
+    "tenantID": "X246M3WS"
+  },
+  "clientResponsePayload": {
+    "orchestrationDecisions": [
+      {
+        "sequenceId": "1",
+        "decisionSource": "PepSanctions01",
+        "decision": "NOTAUTH",
+        "decisionReasons": [
+          "Not Authenticated"
+        ],
+        "score": 0,
+        "decisionText": "Not Authenticated",
+        "nextAction": "Continue",
+        "decisionTime": "2022-06-24T12:36:59Z"
+      }
+    ],
+    "decisionElements": [
+      {
+        "serviceName": "PepSanctions01",
+        "applicantId": "APPLICANT_1",
+        "decision": "NA00",
+        "score": 20,
+        "decisionText": "",
+        "decisionReason": "The Applicant has not been Authenticated to your required minimum level. You should seek o",
+        "appReference": "6FXPN3G9WP",
+        "rules": [
+          {
+            "ruleId": "P1",
+            "ruleName": "PEPs",
```

Comment:
Looks good just wondering if this ruleName is ok/will it need changing? We could change it to be the Pcode we are returning i.e P1 to be inline with the two specific Json responses but not a big issue as this won't be in production and we are only testing against ruleId